src/register/RegistrationPage.jsx

```diff
@@ -424,17 +416,15 @@ const RegistrationPage = (props) => {
 
   const handleOnBlur = (event) => {
     const { name, value } = event.target;
-    if (registrationEmbedded) {
-      if (name === 'name') {
-        validateInput(
-          name,
-          value,
-          { name: formFields.name, username: formFields.username, form_field_key: name },
-          !validationApiRateLimited,
-          false,
-        );
-      }
-      return;
+
+    if (name === 'name') {
+      validateInput(
+        name,
+        value,
+        { name: formFields.name, username: formFields.username, form_field_key: name },
+        !validationApiRateLimited,
+        false,
+      );
```

Comment:
This complete functionality was added for embedded registration. We don't need it for registration page: https://github.com/openedx/frontend-app-authn/pull/950/files

Reply:
addressed